src/lib.rs

```diff
@@ -134,6 +134,21 @@ impl<'input> Document<'input> {
     pub fn text_pos_at(&self, pos: usize) -> TextPos {
         xmlparser::Stream::from(self.text).gen_text_pos_from(pos)
     }
+
+    /// Returns the raw text of the original document.
```

Comment:
I would prefer *input* rather than *raw*. So it will look like: 

Returns the input text that was used to parse the document.

Reply:
Should the name of `raw_text` be changed to `input_text` as well?

Reply:
Yes.